seed/models/analyses.py

```diff
@@ -45,3 +51,14 @@ class Analysis(models.Model):
     # that are applicable to the entire analysis (ie all properties involved).
     # For property-specific results, use the AnalysisPropertyView's parsed_results
     parsed_results = JSONField(default=dict, blank=True)
+
+    def getPropertyViewInfo(self, property_id=None):
+        if property_id is None:
+            analysis_property_views = self.analysispropertyview_set.filter(analysis=self.id)
```

Comment:
I think `self.analysispropertyview_set.filter(analysis=self.id)` should return the same thing as `self.analysispropertyview_set`. Every AnalysisPropertyView can only be linked to a single Analysis

Reply:
If that's the case I will simplify it.  Good to know!